src/components/forms/text-input/text-input.tsx

```diff
@@ -24,6 +24,8 @@ const Input = styled.input<{ hasError?: boolean }>`
 
 export const Label = styled.label`
   ${TextStyles.textXS}
+  font-size: 13px;
```

Comment:
`font-size` shouldn't be set manually, but with the `TextStyles` object. We don't have a text style with 13px, so let's just create a new one (you can copy&paste the `textXS` if that fits).

I just add this comment here, but the new text style should be used everywhere in this PR.